Celeste.Mod.mm/Patches/MapData.cs

```diff
@@ -36,7 +36,9 @@ public patch_MapData(AreaKey area)
             // no-op. MonoMod ignores this - we only need this to make the compiler shut up.
         }
 
+        [PatchTrackableStrawberryCheck]
```

Comment:
Waaaaaait a second, this works?! *checks MonoMod*

[Apparently I've added support for it in 2017 and forgot about it ever since...](https://github.com/MonoMod/MonoMod/commit/5072f2bf772384c8271d524e0d58ef99aef39447)